sdk-utils/src/main/kotlin/com/rakuten/tech/mobile/sdkutils/StaticInfoUtil.kt

```diff
@@ -0,0 +1,191 @@
+package com.rakuten.tech.mobile.sdkutils
+
+import android.content.Context
+import android.os.Build
+import androidx.annotation.RequiresApi
+import org.json.JSONObject
+
+/**
+ * This utility class provides application information and sdk information.
+ */
+object StaticInfoUtil {
+    /**
+     * Get the application information.
+     *
+     * @return the application info
```

Comment:
Mention the return type, and that keys are in `AppInfoKey`.

Reply:
done